x/staking/keeper/val_state_change.go

```diff
@@ -28,6 +28,7 @@ func (k Keeper) ApplyAndReturnValidatorSetUpdates(ctx sdk.Context) (updates []ab
 	store := ctx.KVStore(k.storeKey)
 	maxValidators := k.GetParams(ctx).MaxValidators
 	totalPower := sdk.ZeroInt()
+	amtToNotBonded, amtToBonded := sdk.ZeroInt(), sdk.ZeroInt()
```

Comment:
These variable names are a bit weird. `amtBonded` and `amtNotBonded` would read slightly better and semantically mean the same thing, no?